### ISSUE TITLE
Difference in content when parsing fastq files.

### ISSUE THREAD

**Issue by JasperBoom** [lmdu/pyfastx#15]
Hi, tyvm for `pyfastx`, I really like the it.

I came across a issue when I was writing tests for my script, and hoped you might be
able to help me.

I am processing fastq files, both regular and gzipped using:
```
for oisRecord in pyfastx.Fastq(input_file):
```
And get the headers and sequences like this:
```
oisRecord.name
oisRecord.seq
```

The issue is that I got different read counts for the same fastq file.
When I write the headers and sequences to a file within the for loop, and then
count the lines using `wc -l`, they differ.
For comparison, I also converted the fastq to fasta, and gzipped the fasta.
Both the regular fasta and the gzipped fasta give me the same result as the gzipped fastq.
The outlier is the regular fastq file.

Any idea what could be the cause of this?

System:
`Ubuntu 20.04`
`Python 3.8`
`Pyfastx 0.6.12`

[test.fastq.gz](https://github.com/lmdu/pyfastx/files/4820184/test.fastq.gz) (quality in this fastq is `fake`)

**Comment by lmdu** [comment]
Thank you for reporting this issue. The difference was caused by the error cache buffer. I have fixed in the new version 0.6.13.

**Comment by JasperBoom** [comment]
Thanks!
